src/components/Login/components/authError/AuthError.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import styles from './AuthError.css';
+
+export default class AuthError extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  getErrorMessage(error = {}) {
+    const {
+      code = 'default.error',
+      parameters = [],
+    } = error;
+
+    const values = parameters.reduce((res, { key, value }) => ({ ...res, [key]: value }), {});
+
+    return <FormattedMessage id={`stripes-core.loginServices.login.${code}`} values={values} />;
+  }
+
+  getPasswordValidationMessages(errors) {
+    return errors.map(element => (
+      <li key={`${element.code}-${element.type}`}>
+        {this.getErrorMessage(element)}
+      </li>
+    ));
+  }
+
+  parseErrors(errors) {
+    return errors.length > 1 ? (
+      <ul className={styles.errorsList}>{this.getPasswordValidationMessages(errors)}</ul>
+    ) : (
+      this.getErrorMessage(errors[0])
+    );
+  }
+
+  render() {
+    const { errors } = this.props;
```

Comment:
Why do we need errors in render? All you do is just passing them to parseErrors method. Let's move it to parseErrors.